int64 id
ltlplanner/plan plan
